### PR TITLE
GP Updates - September 2024

### LINKED ISSUE #26487: [INTERNAL] GP Updates
### Describe the issue

Issue to link GP update PRs to.

### Expected behavior

GP Updates merged.

### Steps to reproduce

N/A

### Additional context

_No response_

### I will provide a fix for a bug

- [X] I will provide a fix for a bug

### PR DESCRIPTION
**Pull request by jaymckinney-enavate**
This PR contains the following enhancements to the GP migrations.

- Hide the Run Data Upgrade Now action when the One Step Migration is enabled.
- Remove the dependency on Sales and Inventory Forecast app.
- Obsolete the old GP pages around the staging table data.

Fixes #26487
Fixes [AB#550763](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/550763)

### CODE REVIEW

File: Apps/W1/HybridGP/app/Permissions/HybridGPObjects.PermissionSet.al [hunk 1]
```diff
@@ -141,7 +130,25 @@ permissionset 4029 "HybridGP - Objects"
...
                     page "Hybrid GP Errors Overview Fb" = X,
+#pragma warning restore AL0432
+#endif
+
+#if not CLEAN26
+#pragma warning disable AL0432
+                    page "GP Account" = X,
+                    page "GP Fiscal Periods" = X,
+                    page "GP GLTransactions" = X,
+                    page "GP Customer" = X,
```
Comment (nikolakukrika) [inline Apps/W1/HybridGP/app/Permissions/HybridGPObjects.PermissionSet.al:143]: I think we still need the GP Customers, Vendors and Items, right?

**Review by nikolakukrika — CHANGES_REQUESTED** [review]
We cannot remove the staging tables as they are still actively used. The rest of the changes are fine

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 24886.

**Comment by JesperSchulz** [comment]
Availability update: We will publish a fix for this issue in the next update for release 25.

Build ID to track: 26506.

**Pull request merged**